datasets/open_targets/pipelines/copy_platform_data/pipeline.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+resources: ~
+
+dag:
+  airflow_version: 2
+  initialize:
+    dag_id: copy_platform_data
+    default_args:
+      owner: "Google"
+      depends_on_past: False
+      start_date: '2021-11-23'
+    max_active_runs: 1
+    schedule_interval: "@monthly"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "KubernetesPodOperator"
+      description: "Transfer Open Targets Databases"
+      args:
+        task_id: "copy_bq_datasets"
+        name: "copy_bq_datasets"
+        namespace: "composer"
+        service_account_name: "datasets"
+
+        image_pull_policy: "Always"
+        image: "{{ var.json.open_targets.container_registry.copy_bq_datasets }}"
+        env_vars:
+          SOURCE_PROJECT_ID: "{{ var.json.open_targets.source_project_id }}"
+          TARGET_PROJECT_ID: "{{ var.json.open_targets.target_project_id }}"
+          SERVICE_ACCOUNT: "{{ var.json.open_targets.service_account }}"
+          DATASET_NAME: "open_targets"
+          DATASET_VERSIONS: >-
+            ["platform"]
```

Comment:
See my previous comments. Let's remove this and add parameters that fits better with what the pipeline is expected to do:
```
SOURCE_DATASET
TARGET_DATASET
```

Reply:
@adlersantos done